includes/rest/class-collection.php

```diff
@@ -225,4 +293,25 @@ public static function request_parameters() {
 
 		return $params;
 	}
+
+	/**
+	 * The supported parameters
+	 *
+	 * @return array list of parameters
+	 */
+	public static function request_parameters_for_replies() {
+		$params = array();
+
+		$params['type'] = array(
```

Comment:
You can use an `enum` for that, so that you do not have to check it in the action.

See: https://github.com/Automattic/wordpress-activitypub/blob/8bff92d26020c575782f50740dece4f6c6ed7247/includes/rest/class-followers.php#L134